src/backend/routers/conversation.py

```diff
@@ -351,7 +352,9 @@ async def upload_file(
     """
 
     user_id = ctx.get_user_id()
-    validate_file_size(session, user_id, file)
+    # Currently do not limit file size for Compass
```

Comment:
@GangGreenTemperTatum Pointing this out here.

There isn't really a great way to check do this validation for Compass storage. We decided to just skip this validation for the moment. (GDrive files also does not have this check). Also since files can be uploaded to agents which are shared, we need to think about how that contributes to a user's limit...

Will probably have to do something where we track and store file sizes in the DB in `user` table manually or something

Reply:
https://linear.app/cohereai/issue/TLK-987/investigate-limiting-file-sizes-for-uploads